src/RWXLoader.js

```diff
@@ -236,6 +239,12 @@ function applyTextureToMat( threeMat, folder, textureName, textureExtension = '.
 
 	} ) );
 
+	if ( textureExtension === '.png' ) {
+
+		threeMat.alphaTest = 1;
```

Comment:
Ok so, 1.0 makes textures look sharp (not-blurry). We could probably use this value on masked jpgs as well, but I have no tried this use case.

as for masked png: I have changed the code so that it will look sharp when using a mask on top of a png. It works well.